jsondump/test.c

```diff
@@ -0,0 +1,80 @@
+/*
```

Comment:
can we move this into pytests?
create this C file from a python script and and run it and check exist status

Reply:
Good idea, done (not exactly as suggested).